command/base_predict.go

```diff
@@ -421,16 +447,25 @@ func (p *Predict) policies() []string {
 	return policies
 }
 
-// mounts returns a sorted list of the mount paths for Vault server for
-// which the client is configured to communicate with. This function returns the
-// default list of mounts if an error occurs.
-func (p *Predict) mounts() []string {
+func (p *Predict) mountInfos() (map[string]*api.MountOutput, error) {
```

Comment:
Could you please add a function comment here?

Reply:
done